### ISSUE TITLE
[EN][Spell][paladin] Divine Shield

### ISSUE THREAD

**Issue by Frainzil** [Atlantiss/BugTracker#3049]
For this i'll also link the description of the spell:

Protects you from all damage and spells for 8 sec, but reduces all damage you deal by 50%.  

Cannot be used on a target with Forbearance.  Causes Forbearance for 1 min.

Now if i read this correctly you should NEVER die whenever you activate this spell nor have it up. Now i have noted some different things here that you can die when activating the spell, or even die when bubble is up!?

i Got an example for both of them. 
Now Fafal made a bugtracker issue about this earlier on that on v&t fight people die when they have blackout/iceblock and when they activate divine shield/ice block they just die. I think that bubble/ice block first dispel all possible debuffs before giving you the immune 'phase'. So you actually kill yourself by using an ability that should save you. This happens cause of the scripts you use for this bubble and ice block.

About the 2nd part i have a really good example about what is wrong with the scripts of those 2 spells and what is wrong with the scripts of spells done by bosses. Doing Deadmines hc dungeon we arrived at the last boss; Vanessa VanCleef.
Upon getting this boss to 1% she will use the spell: Powder Explosion:

Powder Explosion Deadly — When Vanessa has only 1% of her total health remaining, she will pull out a final keg of black powder and light it. The explosion kills her, and also deals 185000 to 215000 Fire damage to any enemy target within 10 yards. (http://wow.gamepedia.com/Vanessa_VanCleef_%28tactics%29) 

If i'm not mistaking i read that she deals some dmg to her surrounding area's which isn't that much of a deal. Now since i'm a proud paladin, i use bubble right before the detonation. Unfortunately I die every time i do that. It seems that the bubble just 'disappears'. It also might be that the scripts of the spell just kill everyone around her instead of doing a certain amount of dmg?

**Comment by Varchord** [comment]
Well, it's no exactly like that. 
There are certain spells, damage sources which can bypass the bubble

**Comment by Frainzil** [comment]
What kind of dmg are u talking about?

**Comment by mati865** [comment]
Chaos Bolt for an example http://cata.openwow.com/spell=50796.
"Absorbtion effects are spells like mana shield and power world shield, stuff that absorbs damage. Iceblock and Divine Shield are both immunity effects that grant full immunity to damage, not absorbtion. Your Chaos Bolt should be piercing shields though. " http://us.battle.net/wow/en/forum/topic/2842887043
Also
"In case you havent noticed the description says, Pierces through any absorbsion effects. I dont see immunities anywhere. When i am hit while iceblocked it says immune, not absorb. So yeah, CB will probably pierce through icebarrier, manashield etc, etc, But not through babble or iceblock. As for which spec is pve and which pvp that should probably be posted in another thread." http://www.wowhead.com/forums&topic=34430/warlock-chaos-bolt-1rst-bubble-braker#p312605

However Fury of Frostmourne in WotLK would kill you anyway
"Before 3.3.3 you could survive it with invisibility/vanish/DIing player. You were also able to kill LK during the RP if enough DPS manage to avoid dying (did it just before patch hit live servers with 5 dps alive in 25man). Only thing that happened was that RP ended-->no res-->everyone had to run ).

It now kills DId players and vanish/invisibility doesn't work either. Dying before he kills the raid and having combat res/ankh ready and ressing at the same time he kills the raid might still work but can't confirm it. " http://www.mmo-champion.com/threads/707936-Surviving-Fury-of-the-Frostmourne?p=7613852&viewfull=1#post7613852

So Boss mechanics are designed (or were in WotLK) to kill you anyway.

**Comment by Frainzil** [comment]
Yes that i can understand, there are some spells where you just die, doesn't matter what you will do, any spell you use gets negated by what he does. But it also known that that spell is going to happen and negates anything you use. What i'm talking about is spells where the description clearly describes that the spell is doing a certain amount of dmg, and for example doesn't say: .. will kill everyone in a 20 yard radius of the boss (here no dmg is dealt, you just die if you are in that radius) I totally can understand that using bubble here won't save you. Though both spells i linked don't have such a description, they both say, a certain amount of dmg is dealt to a target or in a certain radius.

**Comment by mati865** [comment]
Fury of Frostmourne "Deals 1,000,000 Shadow damage to all enemies and rendering them unable to release spirit or resurrect. 2.6 sec cast. Cast at the beginning of phase 4" http://www.wowwiki.com/Lich_King_(Icecrown_Citadel_tactics)

As you can see it's not always so oblivious.

**Comment by Frainzil** [comment]
That is true. Though a lot of players now about this spell and know they will die and they don't bother casting the shield anyway since it doesn't matter, you achieved what you wanted to achieve: killing lich king. The spell isn't really comparable either way since you will get a spell later on which resses you, so actually it doesn't matter at all if you would use your bubble, why would you use it if you are getting a ress from Terenas. i'm talking about mid spell fights where you would die cause the scripts of the bubble are killing you, not the mechanic of the boss/the animation after you secured the kill.

**Comment by Glare** [comment]
http://www.wowhead.com/npc=45992/valiona

Check comments for tips from players.

For example this one:
"Several classes have abilities that they can use to eat blackout so you don't need to bother gathering up. Mages can iceblock out of it, pallies can bubble, rogues can cloak of shadows, and shadow priests can use dispersion (spriests will still need to be dispelled by someone else while dispersed, but all the other abilities will auto-dispel the buff when used). The targeted player will need to make sure to be a /range 10 from others, though, as people who are too close will still take damage when the debuff drops."

Another thing worth to mention is that it worked fine around 8 months ago when I was still actively raiding which means the change was made either on transitioning from 4.0.6 to 4.3.4 or in one of the revisions.

**Comment by kadi2** [comment]
Just wondering if you can take the dispell damage just by bubbling Unstable Affliction debuff.

**Comment by mati865** [comment]
@kadi2 Bubble gives you Immune effect to player spells

**Comment by Glare** [comment]
But Unstable Affliction deals dmg when dispeled and that's probably what he meant.

**Comment by kadi2** [comment]
As far as i know you can set Cloak of Shadows, Divine Shield and Ice Block to make these skills remove debuffs instead dispelling them.

**Comment by aneq** [comment]
the problem lies with core purging the character from all the debuffs first and applying immunity after
it should be the other way around. (first immune then dispel)
bubbling a blackout shouldn't kill people

**Comment by kadi2** [comment]
@aneq 
http://us.battle.net/wow/en/forum/topic/2369681101
http://us.battle.net/wow/en/forum/topic/2325683821
Couldn't find any complains about Blackout killing rogues through Cloak of Shadows tho.

**Comment by Jenovas** [comment]
Powder Explosion has a flag - SPELL_UNAFFECTED_BY_INVULNERABILITY so it aint a bug that it deals damage

**Comment by Kejmil** [comment]
http://www.wowhead.com/spells?filter=cr=46;crs=1;crv=0
